internal/cluster/failover_service.go

```diff
@@ -39,9 +39,9 @@ type CandidateCluster struct {
 	ClusterName        string
 }
 
-type addrFun func(*pubcluster.Config, ilogger.Logger) (AddressProvider, AddressTranslator)
+type addrFun func(*pubcluster.Config, logger.Logger) (AddressProvider, AddressTranslator)
```

Comment:
Wer can name import aliases for public packages with `pub` prefix, like `publogger`.

Reply:
This is internal logger package. I could not understand the point. Can you explain further?

Reply:
I don't think we need the internal `logger.Logger` anymore, do we?

Reply:
removed as discussed